packages/block-library/src/quote/deprecated.js

```diff
@@ -87,7 +165,9 @@ const v3 = {
 			</blockquote>
 		);
 	},
-	migrate: migrateToQuoteV2,
+	migrate( attributes ) {
+		return migrateTextAlign( migrateToQuoteV2( attributes ) );
+	},
```

Comment:
```suggestion
	migrate( attributes ) {
		const [ v2Attributes, innerBlocks ] = migrateToQuoteV2( attributes );
		const v4Attributes = migrateTextAlign( v2Attributes );
		return [ v4Attributes, innerBlocks ];
	},
```

This needs to take into account the fact that `migrateToQuoteV2` will return an array containing the attributes and inner blocks. Rather than update all the places where `migrateToQuoteV2` was wrapped, it might be easier to refactor `migrateTextAlign` to accept the array with inner blocks and return them as well.